src/DocumentFormat.OpenXml.Framework/Packaging/OpenSettings.cs

```diff
@@ -77,5 +79,15 @@ public MarkupCompatibilityProcessSettings MarkupCompatibilityProcessSettings
         /// Gets or sets a delegate that is used to create a handler to rewrite relationships that are malformed. On platforms after .NET 4.5, <see cref="Uri"/> parsing will fail on malformed strings.
         /// </summary>
         public Func<OpenXmlPackage, RelationshipErrorHandler>? RelationshipErrorHandlerFactory { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to ignore an exception if the calcChain part is missing.
+        /// The default value is false which means missing calcChain part will throw an exception upon package open.
+        /// </summary>
+        public bool IgnoreExceptionOnCalcChainPartMissing
+        {
+            get => _ignoreExceptionsOnCalcChainPartMissing ?? false;
+            set => _ignoreExceptionsOnCalcChainPartMissing = value;
+        }
```

Comment:
this could be written more succinctly:

```csharp
public bool IgnoreExceptionOnCalcChainPartMissing { get; set; } = false;
```

Reply:
I was just following the existing pattern (i.e. AutoSave). It's probably more readable but we can use either.

Reply:
I prefer using the most modern syntax available. Maybe refactor the autosave to use the c# 6 syntax too

Reply:
I updated the syntax